src/table/Root.tsx

```diff
@@ -24,6 +26,17 @@ export function render(props: RenderProps, reactRoot?: ReactDom.Root) {
   );
 }
 
+export function renderVirtualizedTable(props: WrapperProps, reactRoot?: ReactDom.Root) {
+  const muiTheme = muiSetup('ltr');
+
+  reactRoot?.render(
+    <React.StrictMode>
```

Comment:
FYI we scrapped strict mode since we got douple rerenders from useEffects I think it was

Reply:
https://medium.com/geekculture/the-tricky-behavior-of-useeffect-hook-in-react-18-282ef4fb570a Might be ways around it though

Reply:
I don't really have any issue with it being run in strict mode. It works as intended in dev env. So IMO I might as well keep it to potentially catch issues down the road.

Reply:
okok, seems to be in dev mode as well